packages/ui/src/lib/layouts/Page.svelte

```diff
@@ -47,7 +47,7 @@ function handleKeydown(e: KeyboardEvent): void {
           {/if}
           {#if breadcrumbRightPart}
             <div class="mx-2">&gt;</div>
-            <div class="grow font-extralight" aria-label="name">{breadcrumbRightPart}</div>
+            <div class="grow font-extralight" aria-label="name" role="navigation">{breadcrumbRightPart}</div>
```

Comment:
It looks to me, that navigation could be applied one level up, just after `showBreadCrumb` macro. That div holds back and close button as an navigation elements.

Reply:
And the aria-label `name` is not much of use here.